patches/node/build_add_gn_build_files.patch

```diff
@@ -217,7 +209,7 @@ index 0000000000000000000000000000000000000000..2d8952a1d23af0ab4b3e5de0e56ec1e3
 +    "deps/llhttp",
 +    "deps/nghttp2",
 +    "deps/uvwasi",
-+    "deps/zlib",
++    "//third_party/zlib",
```

Comment:
Why is this changed ? This was added for https://github.com/electron/node/pull/75

Reply:
@deepak1556 Node.js itself switch to Chromium's zlib - they're the same now.

Reply:
Thanks @codebytere , that context was missing in the commit :)

Ping @adill we should export symbols from chromium's zlib now https://github.com/electron/electron/pull/15103 , we no longer support component build so the reason for original revert is void.